File: SEP3-TIER2/src/Server/Model/SubTask.java
package Server.Model;

public class SubTask {
    private int taskid;
    private int id;
    private boolean iscomplete;
    private String description;
    private String starttime;
    private String endtime;
    private String colorlabel;

    public SubTask() {
    }

    public int getTaskid() {
        return taskid;
    }

    public void setTaskid(int taskid) {
        this.taskid = taskid;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public boolean isIscomplete() {
        return iscomplete;
    }

    public void setIscomplete(boolean iscomplete) {
        this.iscomplete = iscomplete;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getStarttime() {
        return starttime;
    }

    public void setStarttime(String starttime) {
        this.starttime = starttime;
    }

    public String getEndtime() {
        return endtime;
    }

    public void setEndtime(String endtime) {
        this.endtime = endtime;
    }

    public String getColorlabel() {
        return colorlabel;
    }

    public void setColorlabel(String colorlabel) {
        this.colorlabel = colorlabel;
    }

    @Override
    public String toString() {
        return "SubTask{" +
                "taskid=" + taskid +
                ", id=" + id +
                ", iscomplete=" + iscomplete +
                ", description='" + description + '\'' +
                ", starttime='" + starttime + '\'' +
                ", endtime='" + endtime + '\'' +
                ", colorlabel='" + colorlabel + '\'' +
                '}';
    }
}

File: SEP3-TIER1/BlazorTest/Model/SubTask.cs
using System;
using Newtonsoft.Json;

namespace BlazorTest.Model
{
    public class SubTask
    {
        [JsonProperty("taskid", NullValueHandling = NullValueHandling.Ignore)]
        public int TaskId { get; set; }

        [JsonProperty("id", NullValueHandling = NullValueHandling.Ignore)]
        public int Id { get; set; }

        [JsonProperty("iscomplete", NullValueHandling = NullValueHandling.Ignore)]
        public bool IsComplete { get; set; }

        [JsonProperty("description", NullValueHandling = NullValueHandling.Ignore)]
        public string Description { get; set; }

        [JsonProperty("starttime", NullValueHandling = NullValueHandling.Ignore)]
        public DateTime StartTime { get; set; }

        [JsonProperty("endtime", NullValueHandling = NullValueHandling.Ignore)]
        public DateTime EndTime { get; set; }

        [JsonProperty("colorlabel", NullValueHandling = NullValueHandling.Ignore)]
        public string ColorLabel { get; set; }
    }
}